File: wasm3/zig/examples/r2pipe.zig
const std = @import("std");

extern "r2" fn add(a: i32, b: i32, mul: *i32) i32;

fn main() void {
    const add_res = add(a1, a2, &mul_res);
    std.info.log("Hello World", .{add_res});
}

File: wasm3/zig/examples/r2pipe.c
#include <stdio.h>

//__attribute__((export_name("r2.add")))
extern int add (int a, int b);

__attribute__((export_name("main")))
int main() {
	// printf ("Hello World %d\n", add (1, 2));
	printf ("Hello World %d\n", 123);
}
